examples/videos/deno_coverage.md

```diff
@@ -4,3 +4,39 @@ url: /examples/deno_coverage/
 videoUrl: https://www.youtube.com/watch?v=P2BBYNPpgW8
 layout: video.tsx
 ---
+
+## Description of video
+
+We updated `deno coverage` in 1.39 with a better output and HTML generation.
+
+## Transcript and code
+
+If you're using Deno test, have you checked out Dino coverage?
+
+Dino coverage is a great way to see how much test coverage you have, just add
```

Comment:
not sure if we should always refer to the feature using with lower case and monospace for the command or not, but adding that as I fix the Dino thing here

```suggestion
`deno coverage` is a great way to see how much test coverage you have, just add
```